sklearn/feature_selection/_univariate_selection.py

```diff
@@ -334,6 +334,14 @@ def r_regression(X, y, *, center=True, force_finite=True):
     return correlation_coefficient
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "y": ["array-like"],
+        "center": [bool],
+        "force_finite": [bool],
```

Comment:
```suggestion
        "center": ["boolean"],
        "force_finite": ["boolean"],
```

Reply:
There should be consistency in the parameter validation some have used bool.